CONTRIBUTING.md

```diff
@@ -1,66 +1,97 @@
+If the issue only reproduces with the JS SDK, you can [open an issue](https://github.com/parse-community/parse-server/issues) on this repository.
 
 Details are key. The more information you provide us the easier it'll be for us to debug and the faster you'll receive a fix. Some examples of useful tidbits:
 
 * A description. What did you expect to happen and what actually happened? Why do you think that was wrong?
-* A simple unit test that fails. Refer [here][tests-dir] for examples of existing unit tests. See our [README](README.md#usage) for how to run unit tests. You can submit a pull request with your failing unit test so that our CI verifies that the test fails.
+* A simple unit test that fails. Refer [here][tests-dir] for examples of existing unit tests and [here][integration-test-dir] for integration tests examples. See for how to setup your machine and run unit tests in [this](#setting-up-the-project-for-debugging-and-contributing) guide. You can submit a pull request with your failing unit test so that our CI verifies that the test fails.
 * What version does this reproduce on? What version did it last work on?
 * [Stacktrace or GTFO][stacktrace-or-gtfo]. In all honesty, full stacktraces with line numbers make a happy developer.
 * Anything else you find relevant.
 
 ### Security Bugs
-Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe disclosure of security bugs. In those cases, please go through the process outlined on that page and do not file a public issue.
+Parse Community has a [responsible Vulnerability Disclosure Program](https://github.com/parse-community/parse-server/blob/master/SECURITY.md) for the safe disclosure of security bugs. In those cases, please go through the process outlined on that page and do not file a public issue.
 
 ## Coding Style
 * Most importantly, match the existing code style as much as possible.
 * We use [Flow](http://flowtype.org/) and ES6 for this codebase. Use modern syntax whenever possible.
 * Keep lines within 80 characters.
 * Always end lines with semicolons.
 
+### Code of Conduct
+
+This project adheres to the [Contributor Covenant Code of Conduct](https://github.com/parse-community/parse-server/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to honor this code.
+
 ## License
 By contributing to the Parse JavaScript SDK, you agree that your contributions will be licensed under its license.
 
  [google-group]: https://groups.google.com/forum/#!forum/parse-developers
- [stack-overflow]: http://stackoverflow.com/tags/parse.com
- [bug-reports]: https://www.parse.com/help#report
- [rest-api]: https://www.parse.com/docs/rest/guide
+ [stack-overflow]: http://stackoverflow.com/tags/parse-server
+ [bug-reports]: https://github.com/parse-community/parse-server/issues
+ [rest-api]: https://docs.parseplatform.org/rest/guide
  [parse-api-console]: http://blog.parse.com/announcements/introducing-the-parse-api-console/
```

Comment:
The link should be http://blog.parseplatform.org/announcements/introducing-the-parse-api-console/